### ISSUE TITLE
How to handle develop branches

### ISSUE THREAD

**Issue by AiraYumi** [secondlife/viewer#1986]
### Description

I don't know how to handle the develop branch.
Will a release branch be created from the develop branch, and then merged into the main branch once the release branch is complete?

What will happen to the existing webrtc-voice, maint-b, and maint-c branches?

**Comment by brad-linden** [comment]
Hello AiraYumi,

Yes, releases will be periodically be created beginning as a snapshot and stabilizing for release from there.  We are migrating to use a "Git Flow" based process for managing our releases from now on.  Our first attempt at this in progress in the branch `release/2024.06-atlasaurus`.  Once a release is promoted to be the default release it will merge into `main`.  

Unfortunately the other branches you list were created before we started using this process, so they don't quite fit yet.  I'll describe our current plan here, hopefully we can make it happen over the coming weeks and months:  `release/webrtc-voice` will proceed and be promoted with the old release process first.  Once that has merged to main, we expect the next release will be `release/2024.06-atlasaurus`.   Next we plan to merge maint-b into `develop` and shortly after that create a new `release/2024.08-...` branch for the next release.  And then do something similar for maint-c in the future.

Sorry if this is unclear while we are migrating, but the hope is that once we get these straggler old release branches out of the way, everything will be much clearer, and the normal development flow will be that PRs always just go into `develop`, and releases always just start as a date based snapshot of `develop`.

hope this helps.
